src/client/api/pythonApi.ts

```diff
@@ -316,6 +317,19 @@ export class PythonInstaller implements IPythonInstaller {
             });
         }
     }
+
+    public async isProductVersionCompatible(
+        product: Product,
+        semVerRequirement: string,
+        resource?: InterpreterUri
+    ): Promise<ProductInstallStatus> {
+        try {
+            const api = await this.apiProvider.getApi();
+            return await api.isProductVersionCompatible(product, semVerRequirement, resource);
+        } catch (ex) {
```

Comment:
If we're not doing anything with the exception, then ther'es no need to capture this.